src/main/java/com/rackspace/ceres/app/web/QueryController.java

```diff
@@ -45,11 +47,16 @@
 
   private final QueryService queryService;
   private final DownsampleProperties downsampleProperties;
+  private final Counter aggregatorQueryCounter;
+  private final Counter noAggregatorQueryCounter;
 
   @Autowired
-  public QueryController(QueryService queryService, DownsampleProperties downsampleProperties) {
+  public QueryController(QueryService queryService, DownsampleProperties downsampleProperties,
+      MeterRegistry meterRegistry) {
     this.queryService = queryService;
     this.downsampleProperties = downsampleProperties;
+    noAggregatorQueryCounter = meterRegistry.counter("Ceres.QueryApi", "Query", "Without_Aggregator");
```

Comment:
name this one as `rawQueryCounter` and `meterRegistry.counter("ceres.query", "type", "raw")`